examples/ft_transformer_text.py

```diff
@@ -51,21 +52,143 @@ def __call__(self, sentences: List[str]) -> Tensor:
 parser.add_argument('--lr', type=float, default=0.0001)
 parser.add_argument('--epochs', type=int, default=100)
 parser.add_argument('--seed', type=int, default=0)
+parser.add_argument('--finetune', action='store_true')
+parser.add_argument('--lora', action='store_true')
```

Comment:
Can we add a warning/error/comment if somebody activates lora but not fine-tune? In that case lora flag is useless.